source/client-side-encryption/tests/basic.template.yml

```diff
@@ -0,0 +1,42 @@
+database_name: &database_name "default"
+collection_name: &collection_name "default"
+
+encryption_setup:
+  collections:
+    - database: default
+      collection: default
+      data: []
+      jsonSchema: {{get_schema(0)}}
+    - database: admin
+      collection: datakeys
+      data: [{{get_key(0)}}]
+tests:
+  - description: "Insert a document requiring encryption" 
+    clientOptions:
+      clientSideEncryptionOptions:
+        kmsProvider: "aws-kms"
+        useRemoteSchemas: True
+    operations:
+      - name: insertOne
+        arguments:
+          document: &doc0 { _id: 1, ssn: "457-55-5462" }
+      - name: findOne
+        arguments:
+          filter: { _id: 1 }
+        result: *doc0
+    expectations:
+      - command_started_event:
+          command:
+            insert: *collection_name
+            documents:
+              - &doc0_encrypted { _id: 1, ssn: {{get_ciphertext("457-55-5462", schema=0, field="ssn")}} }
+            ordered: true
+          command_name: "insert"
+      - command_started_event:
+          command:
+            find: *collection_name
+            filter: { _id: 1 }
```

Comment:
I realize you're probably going to change the operation to use `find` instead of `findOne` (per Jeff's earlier comment about the CRUD spec). If not, though, I wonder if it would have made sense to assert `limit` here.